src/Avatar/index.stories.tsx

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable react/prop-types */
 import React from "react";
 import Avatar from "./";
 import Row from "../Row";
 
-export const Overview = () => {
-  return <Avatar initials="CP" label="Christian Paz" />;
+export const Overview = ({ initials = "CP", label = "Christian Paz", ...otherProps }) => {
+  return <Avatar initials={initials} label={label} {...otherProps} />;
```

Comment:
Passing props through arguments of `Overview` enables controls in the props table.